### PR TITLE
add iD editor

### PR DESCRIPTION
**Pull request by ansis**
Adds [iD](http://ideditor.com) as an editor. 

Opening this now, but merging should wait until iD switches over to the real db. It should be happening very soon.

/cc @tmcw @jfirebaugh no harm in this, right?

### CODE REVIEW

**Comment by ansis** [comment]
@mvexel iD alpha2 release and switched over to live db. Ready to be merged

File: client/js/maproulette.js [hunk 1]
```diff
@@ -190,20 +190,25 @@ function openIn(editor) {
...
 		});
 	} else if (editor == 'p') { // potlatch
 		var PotlatchURL = 'http://www.openstreetmap.org/edit?editor=potlatch2&bbox=' + map.getBounds().toBBoxString();
 		window.open(PotlatchURL);
-		setTimeout("confirmRemap('p')", 4000)
-	}
+		setTimeout("confirmRemap('Potlatch')", 4000)
+	} else if (editor == 'i') { // iD editor
+        var id = currentNodeId ? 'n' + currentNodeId : 'w' + currentWayId,
+            loc = map.getZoom() + '/' + map.getCenter().lat + '/' + map.getCenter().lng;
```
Comment (tmcw) [inline client/js/maproulette.js:203]: This should be switched to

``` js
loc = map.getZoom() + '/' + map.getCenter().lng + '/' + map.getCenter().lat;
```

Before merge.

Comment (emacsen) [inline client/js/maproulette.js:203]: This code branch is done in favor of the new maproulette-front (which is likely to be renamed maproulette at some point). That branch already has iD support but I'll change it in favor of this new logic.

Comment (ansis) [inline client/js/maproulette.js:203]: awesome, thanks. If you could also replace http://geowiki.com/iD with http://openstreetmap.us/iD/release in the one, that would be great

Comment (tmcw) [inline client/js/maproulette.js:203]: @emacsen Where is maproulette-front? Can this repository point to it so that people don't get punked?

**Comment by mvexel** [comment]
yes, will create a clear reference.

**Comment by jfirebaugh** [comment]
Current maproulette.org doesn't have an iD button... what would it take to get that deployed?

**Comment by rhewitt22** [comment]
Bump -- I'd be much more inclined to use maproulette.org if it had the iD editor as an option.

**Comment by mvexel** [comment]
Yea, this is overdue, I know --- we decided to put all our (limited) time
into the next gen maproulette, over at osmlab/maproulette - this will be
ready for release in a couple weeks I hope? then iD will be supported there.

On Fri, Dec 6, 2013 at 5:51 PM, Roy Hewitt notifications@github.com wrote:

> Bump -- I'd be much more inclined to use maproulette.org if it had the iD
> editor as an option.
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/mvexel/remapatron/pull/11#issuecomment-30042706
> .

## 

Martijn van Exel
http://oegeo.wordpress.com/
http://openstreetmap.us/

**Pull request closed without merging**